src/app/main/main.component.js

```diff
@@ -40,6 +49,21 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       url: "/checkout.html",
       template: '<checkout></checkout>'
     })
+    .state('profile', {
+      url: "/profile",
+      abstract: true,
```

Comment:
ui-router is only included in our development environment, so we can't use it to enforce or resolve state changes. It's kludgy, but in production, each component in /app is actually an independent angular application and not a single application like we have in development (This is a design requirement of Adobe Experience Manager and analytics). This also requires that state management is handled inside each base component (parent/child ui-router states won't work in production).